lib/mixpanel-node.js

```diff
@@ -236,10 +235,7 @@ var create_client = function(token, config) {
                 batch = batch.map(function (event) {
                     var properties = event.properties;
 
-                    if (endpoint === '/import' || event.properties.time) {
-                        // usually there will be a time property, but not required for `/track` endpoint
-                        event.properties.time = ensure_timestamp(event.properties.time);
-                    }
+                    event.properties.time = ensure_timestamp(event.properties.time || Date.now());
```

Comment:
Unless I'm misreading, this seems to be a behavior change (here and on L294 below). Previously, if you didn't explicitly set a time for a `track()` call, it wouldn't include the property, and the API server would use the time received as the event timestamp. Now that case will result in the client's clock being used instead, which might be something totally weird for example for an IoT device (one of the ways people use this SDK).

I'd prefer to keep the old way, as it offers the client the flexibility not to include the `time` prop.

Reply:
You're correct. The change would bring it more in line with the other SDK's but I hadn't considered the IoT-style use-case where you'd prefer our backend timestamp to the clock on some random toaster. I was just thinking you'd prefer your server's timestamp to ours. I'll revert it back.